merlin/server/server_config.py

```diff
@@ -0,0 +1,105 @@
+import logging
+import os
+
+import yaml
+
+
+LOG = logging.getLogger("merlin")
+
+MERLIN_CONFIG_DIR = os.path.expanduser("~") + "/.merlin/"
```

Comment:
These values are defined in [configfile.py](https://github.com/LLNL/merlin/blob/0497a98f5d29ef0047b1db39ff362696df7d79e5/merlin/config/configfile.py#L50-L51), but I'm not able to import this value since there is a [get_config()](https://github.com/LLNL/merlin/blob/0497a98f5d29ef0047b1db39ff362696df7d79e5/merlin/config/configfile.py#L305-L306) call at the end of configfile.py, which requires the ~/.merlin directory to exist unless it will error. I noticed that in other places the configuration directory is locally defined such as here in the [main.py](https://github.com/LLNL/merlin/blob/0497a98f5d29ef0047b1db39ff362696df7d79e5/merlin/main.py#L312-L313) file, so I defined it globally in the server_config.py file.